### DISCUSSION TITLE
typo in installation "inculde"

### DISCUSSION THREAD

**Discussion by sebfeldm** [Rocket-Search/sap-mon#2] [General]
web_srv.cpp:17:10: fatal error: /usr/inculde/curl/curl.h: No such file or directory
   17 | #include "/usr/inculde/curl/curl.h"

**Comment by Rocket-Search** [comment]
Hello seebus,

thank you for you reply.
Please install curl headers on you system or adjust the path in the code to you local curl headers.
If you don`t have curl installed, curl can be found here https://curl.se/download.html 
or you can install curl from your linux distribution repository server .

Best regards
T. Reineck

**Reply by sebfeldm** [reply]
Thanks! Yes you are correct, I just wanted to point out the typo in your web_srv.cpp ;)

**Comment by Rocket-Search** [comment]
fixed
